README.md

```diff
@@ -44,10 +44,19 @@
  <img src="https://raw.github.com/bastienwirtz/homer/main/docs/screenshot.png" width="100%">
 </p>
 
+/**
+ * @file README.md
+ * @brief Table of Contents for the Homer project.
+ *
+ * This file contains the table of contents for the Homer project's README.md file.
+ * It provides an overview of the different sections and links to the corresponding documentation files.
+ */
+
```

Comment:
I'm not sure if it's supposed to be displayed or not but I can see on your [repo](https://github.com/jplanckeel/homer/blob/docs/update-kubernetes-installation/README.md) that it is rendered kinda broken. 

I don't think this part is necessary, could you remove it? Thanks!

Reply:
sorry it's an involuntary addition, i have remove it ;)